installer/opensuse/bootstrap

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh -e
+# Copyright (c) 2016 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This is a distro-specific bootstrap script, sourced from main.sh, and as such
+# has access to all of the variables set by main.sh, namely $tmp (the temporary
+# directory), $INSTALLERDIR/$DISTRO, $RELEASE, $BOOTSTRAP_RELEASE (if different
+# from $RELEASE), $ARCH, and $MIRROR.
+
+# Tumbleweed
+url_tumbleweed_armv7hl="http://download.opensuse.org/ports/armv7hl/tumbleweed/images/openSUSE-Tumbleweed-ARM-JeOS.armv7hl-rootfs.armv7l-Current.tbz"
+url_tumbleweed_aarch64="http://download.opensuse.org/ports/aarch64/tumbleweed/images/openSUSE-Tumbleweed-ARM-JeOS.aarch64-rootfs.aarch64-Current.tbz"
+
+# Leap
+url_leap_42_2_armv7hl="http://download.opensuse.org/ports/armv7hl/distribution/leap/42.2/appliances/openSUSE-Leap42.2-ARM-X11.armv7-rootfs.armv7l-2017.02.02-Build1.1.tbz"
+url_leap_42_2_aarch64="http://download.opensuse.org/ports/aarch64/distribution/leap/42.2/appliances/openSUSE-Leap42.2-ARM-JeOS.aarch64-rootfs.aarch64-2017.02.02-Build1.1.tbz"
+
+BOOTSTRAP_ARCH="$ARCH"
+BOOTSTRAP_RELEASE="$RELEASE"
+
+# Rename factory to tumbleweed
+[ "$RELEASE" = "factory" ] && BOOTSTRAP_RELEASE=tumbleweed
+# Fix leap release name
+[ "$RELEASE" = "leap-42.2" ] && BOOTSTRAP_RELEASE=leap_42_2
+# fix up the good old toolchain/kernel naming conflict
+[ "$ARCH" = "arm64" ] && BOOTSTRAP_ARCH=aarch64
+
+key=$(echo "\$url_${BOOTSTRAP_RELEASE}_${BOOTSTRAP_ARCH}")
```

Comment:
I *think* these two lines can be replaced with:

`eval "URL=\"\$url_${BOOTSTRAP_RELEASE}_${BOOTSTRAP_ARCH}\""`

Reply:
or to handle the nonexistent case better:

`eval "URL=\"\${url_${BOOTSTRAP_RELEASE}_${BOOTSTRAP_ARCH}-}\""`

which just sets URL to an empty string.

Reply:
yes, will do so.